source/openapi3/split.yaml

```diff
@@ -51,7 +51,7 @@ info:
     * Dates & times are returned in UTC using [ISO
     8601](https://en.wikipedia.org/wiki/ISO_8601) format with second accuracy.
     With requests, when no TZ is supplied, the configured TZ of the
-    authenticated user is used, or `Australia/Sydney` if no TZ is configured.
+    authenticated user is used, or `Pacific/Auckland` if no TZ is configured.
```

Comment:
Very weird that these changes are showing even though they're in `main` branch already

Reply:
> Very weird that these changes are showing even though they're in `main` branch already

because the branch has not been updated with the main branch yet. Normally you would merge main back to your branch or rebase it